ui/src/components/denali/icons/Icon.js

```diff
@@ -22,9 +22,10 @@ const Icon = (props) => {
     const icon = ICONS[props.icon];
     const width = props.size ? props.size : props.width;
     const height = props.size ? props.size : props.height;
+    const viewBoxDimensions = '0 0 ' + props.viewBoxWidth + ' ' + props.viewBoxHeight;
     return (
         <svg
```

Comment:
Changes in Icon.js:
1. Provide the option to set icon size - denali icons did not appear when the viewBox was set to defaults.